FirstFactorial.java

```diff
@@ -0,0 +1,21 @@
+import java.util.*; 
+import java.io.*;
+
+class FirstFactorial {
+
+  public static int FirstFactor(int num) {
+    
+    if (num <= 1){ 

```

Comment:
How do you handle negative values here?  For example -100 factorial would return 1, would that be correct?

Reply:
Actually the code can only be used for numbers greater than or equals to one

Reply:
i do not think the answer would be correct for -100
i used this code for a test in my 200level and i got my mark so i've been using it since then

Reply:
Okay, can you consider negative numbers?